projects/plugins/jetpack/modules/likes.php

```diff
@@ -76,20 +84,24 @@ public function __construct() {
 		add_action( 'jetpack_activate_module_likes', array( $this, 'set_social_notifications_like' ) );
 		add_action( 'jetpack_deactivate_module_likes', array( $this, 'delete_social_notifications_like' ) );
 
-		Jetpack::enable_module_configurable( __FILE__ );
+		if ( $this->in_jetpack ) {
+			Jetpack::enable_module_configurable( __FILE__ );
+		}
+
 		add_filter( 'jetpack_module_configuration_url_likes', array( $this, 'jetpack_likes_configuration_url' ) );
 		add_action( 'admin_print_scripts-settings_page_sharing', array( $this, 'load_jp_css' ) );
 		add_filter( 'sharing_show_buttons_on_row_start', array( $this, 'configuration_target_area' ) );
 
-		$active = Jetpack::get_active_modules();
+		$publicize_active  = Jetpack::is_module_active( 'publicize' );
+		$sharedaddy_active = Jetpack::is_module_active( 'sharedaddy' );
```

Comment:
The `is_module_active()` makes things more readable, thanks!

However, won't these `Jetpack::` static calls die if the `Jetpack` class doesn't exist? Should we wrap them in a `$this->in_jetpack` check?

Reply:
The "in_jetpack" seems like a bit of a misnomer to me here. Unless we're planning on splitting this out into a separate package for some reason, the real distinction instead seems to be "is the `Jetpack` class the normal one or Simple's stubby version?".